services/thermal_mgr/thermal_mgr.c

```diff
@@ -7,6 +7,8 @@
 #include <os_task.h>
 #include <os_queue.h>
 
+#include <logging.h>
```

Comment:
We're not super consistent about this, but we tend to keep the angle bracket includes for standard libs and the hal/freertos files.
```suggestion
#include "logging.h"
```